distributor-api-v1/pricing.md

```diff
@@ -0,0 +1,132 @@
+# Pricing
+
+* **Net price** is an amount without taxes.
+* **Gross price** is a net price + taxes.
+
+In order to use pricing in the Distributor, the API returns the [Amount](Amount) object as part of responses.
+[Amount](Amount) represents a structure that holds gross price, net price and tax values.
+
+The [Amount](Amount) has following structure:
+
+```javascript
+Currency:
```

Comment:
This should be enclosed in `{ }` too. Instead of "Currency" use either example currency (ie. "USD" for example), or mark it somehow to ensure it's a placeholder for actual value.

Reply:
changed, used specific "USD" example without placeholder